core/include/config.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2019 Pengutronix
+ * All rights reserved.
+ *
+ * Rouven Czerwinski <entwicklung@pengutronix.de>
+ */
+#ifndef __CONFIG_H_
+#define __CONFIG_H_
+
+#define __stringify(s) #s
+
+#define IS_CONFIGURED(feat)			\
+(__stringify(feat) == "y")
```

Comment:
CFG_ variables inside C code are defined as `1` for `y`, see `<out>/include/generated/conf.h`

Reply:
Ah, I'll change that to "1" then.